src/tjota_db.erl

```diff
@@ -84,14 +85,63 @@ create_table(user) ->
         )
     ", [?KEYSPACE, ?TABLE_USER]));
 
-create_table(session) -> not_implemented;
-create_table(room) -> not_implemented;
-create_table(message) -> not_implemented;
+create_table(session) -> 
+    {ok, Client} = get_cqerl_client(),
+    {ok, _} = cqerl:run_query(Client, io_lib:format("
+        CREATE TABLE IF NOT EXISTS ~s.~s (
+        )
+    ", [?KEYSPACE, ?TABLE_SESSION]));
+
 
-create_table(user_room) -> not_implemented;
-create_table(room_user) -> not_implemented.
+create_table(room) -> 
+    {ok, Client} = get_cqerl_client(),
+    {ok, _} = cqerl:run_query(Client, io_lib:format("
+        CREATE TABLE IF NOT EXISTS ~s.~s (
+            room-name TEXT,
+            id UUID,
+            users TEXT,
+            type TEXT,
+            data BLOB,
+            PRIMARY KEY (id)
+        )
+    ", [?KEYSPACE, ?TABLE_ROOM]));
 
 
+create_table(message) ->
+  {ok, Client} = get_cqerl_client(),
+    {ok, _} = cqerl:run_query(Client, io_lib:format("
+        CREATE TABLE IF NOT EXISTS ~s.~s (
+            data TEXT,
+            room_id UUID,
+            sender_id UUID,
+            timestamp TIMESTAMP,
+            PRIMARY KEY (data)
```

Comment:
The primary key for message must be (room_id, timestamp) to perform clustering on the timestamp.
The first column in the primary key `room_id` is the partition key, the remaining columns like `timestamp` are the clustering keys. Other than that the statement is fine.